code/modules/medical/pathology/pathogen_tools.dm

```diff
@@ -1,7 +1,7 @@
 // A blood slide, used by the centrifuge.
 /obj/item/bloodslide
 	name = "Blood Slide"
-	icon = 'icons/obj/pathology.dmi'
+	icon = 'icons/obj/machinery/pathology.dmi'
```

Comment:
This is a good example of why grouping into `machinery` isn't as good as grouping by the things being related; blood sides aren't machinery, but with this change either a) blood slide icons are in `icons/obj/machinery/pathology` or b) blood slide icons are split out from the rest of pathology stuff.

To me that seems less useful.